bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -519,20 +553,15 @@ public void mouseClicked(final MouseEvent e) {
         dragSource = new TelescopePosTableDragSource(this);
     }
 
-    public void telescopePosUpdate(final WatchablePos tp) {
+    @Override public void telescopePosUpdate(final WatchablePos tp) {
         final int index = getSelectedRow();
         _resetTable(_env);
 
         // Restore the selection without firing new selection events.
-        if ((index >= 0) && (index < getRowCount())) {
-            final boolean ignore = _ignoreSelection;
-            try {
-                _ignoreSelection = true;
-                _setSelectedRow(index);
-            } finally {
-                _ignoreSelection = ignore;
-            }
-        }
+        final boolean tmp = _ignoreSelection;
+        _ignoreSelection = true;
+        _setSelectedRow(index);
```

Comment:
I really didn't see where this could throw an exception outside of `JTable.getSelectionModel` returning `null`, which only should happen if row selection is not allowed (which should never be the case).

I'll restore it in the interests of safety, though.